guides/admin-guide.rst

```diff
@@ -1082,23 +1095,20 @@ EXPEDITE or HOLD.
 Canceling jobs
 --------------
 
-An active job may be canceled via the ``flux job cancel`` command. An
+An active job may be canceled via the ``flux cancel`` command. An
 instance owner may cancel any job, while a guest may only cancel their
 own jobs.
 
-All active jobs may be canceled with ``flux job cancelall``. By default
-this command will only print the number of jobs that would be canceled.
-To force cancellation of all matched jobs, the ``-f, --force`` option must
-be used:
+All active jobs may be canceled with ``flux cancel --user=all``.
 
 .. code-block:: console
 
- $ flux job cancelall
- flux-job: Command matched 5 jobs (-f to confirm)
- $ flux job cancelall -f
- flux-job: Canceled 5 jobs (0 errors)
+ $ flux cancel --user=all --dry-run
+ flux-cancel: Would cancel 3 jobs
+ $ flux cancel --user=all
```

Comment:
Just to double check - this doesn't require the `-f` for force?

Reply:
Right.  Now we have `--dry-run` to show what the command is going to do without doing it.  The default is to do stuff now.

Reply:
oh nice! That is a a _hugely_ better design. +1 !